### ISSUE TITLE
Remove classnames dependency

### ISSUE THREAD

**Issue by levbrie** [roylee0704/react-flexbox-grid#49]
I would love to see the `classnames` dependency removed. In general, it seems to me to be best practice to try to encapsulate the dependencies for a given project within the project itself and not to force package users to reproduce your development setup and to use the styling tools you've chosen.  I love this package and what you've done here and think that the structure is really great, but we take great pains not to include packages and build configuration that don't actually apply to our own code - it adds confusion for new developers and it makes it far more difficult to uninstall single dependencies.  Any chance you're planning to or would consider planning to get rid of this dependency?

**Comment by roylee0704** [comment]
sounds good to me!

**Comment by loicginoux** [comment]
Hey, just want to add something, I just tried this package and have the following error which i guess is related to the comment above. 
```ERROR in ./~/react-flexbox-grid/lib/components/Row.js
Module not found: Error: Can't resolve 'classnames' in '/Users/loic/Dropbox/projects/client/alerti/alerti-front/node_modules/react-flexbox-grid/lib/components'
 @ ./~/react-flexbox-grid/lib/components/Row.js 10:18-39```

He complains about `classnames` not found, because it's not a package installed in my project.
Should dependencies not be encapsulated in your package ? 
(After installing classnames in my project, I have the same error with the package `flexboxgrid` that needed to be installed in my project too)

**Comment by bguzryanto** [comment]
@roylee0704 are you working on it? I have similar problem, do think this problem because classnames, flexboxgrid does not present on 'dependecy' in package.json?

**Comment by silvenon** [comment]
Fixed with #66.
